### ISSUE TITLE
[FEATURE] Custom artist profile pictures

### ISSUE THREAD

**Issue by crackheadakira** [geoffrey45/alice-core#14]
This would be helpful for the cases where the app was unable to find an image, or came from a different artist of the same name.

**Comment by cwilvx** [comment]
@crackheadakira 
I agree. This feature will be implemented in future, but I can't promise it will be in the next release. I'll build it when the features planned for the next release are done.

Thanks for the feature request.
